test/models/promise.js

```diff
@@ -3,9 +3,17 @@ import { expect } from 'chai'
 import { promiseGallerySort } from '../../models/promise'
 
 describe('promiseGallerySort', () => {
+  const x = new Date()
```

Comment:
i would use prefer using moment here, and this is what i meant about the bdd/lazy vars (similar to rspec - https://github.com/stalniy/bdd-lazy-var):

```
  def('lastWeek', () => moment().subtract(1, 'weeks'))
  def('yesterday', () => moment().subtract(1, 'days'))
  def('tomorrow', () => moment().add(1, 'days'))
  def('weekFromTomorrow', () => $tomorrow.add(7, 'days')) // same as 1, 'weeks'

  def('promises', () => [{
    tini: $lastWeek,
    tfin: $yesterday,
  }, {
    tini: $lastWeek,
    tfin: null,
  }])

  it('sorts completed promises after pending promises', () => {
    expect(promiseGallerySort(...$promises) > 0).to.be.true
  })
```